apps/ui/src/views/Space/Turbo.vue

```diff
@@ -0,0 +1,330 @@
+          custom: ICInfinity
+        },
+        {
+          title: 'Proposal character limit',
+          basic: limits.value['space.default.body_limit'],
+          turbo: limits.value['space.turbo.body_limit'],
+          custom: ICInfinity
+        }
+      ]
+    },
+    choices: {
+      title: 'Voting',
+      features: [
+        {
+          title: 'Choices',
+          basic: limits.value['space.default.choices_limit'],
+          turbo: limits.value['space.turbo.choices_limit'],
+          custom: ICInfinity
+        },
+        {
+          title: 'Voting strategies',
+          basic: limits.value['space.default.strategies_limit'],
+          turbo: limits.value['space.turbo.strategies_limit'],
+          custom: ICInfinity
+        }
+      ]
+    },
+    tools: {
+      title: 'Management tools',
+      features: [
+        {
+          title: 'Delegates dashboard',
+          basic: '-',
+          turbo: true,
+          custom: true
+        }
+      ]
+    },
+    customization: {
+      title: 'Support and customization',
+      features: [
+        {
+          title: 'Whitelabel',
+          basic: '-',
+          turbo: true,
+          custom: true
+        },
+        {
+          title: 'Priority support',
+          basic: '-',
+          turbo: true,
+          custom: true
+        },
+        {
+          title: 'Early access to new features',
+          basic: '-',
+          turbo: true,
+          custom: true
+        },
+        {
+          title: 'Custom interface',
+          basic: '-',
+          turbo: '-',
+          custom: true
+        }
+      ]
+    }
+  };
+});
+
+function toggleQuestion(id: number) {
+  currentQuestion.value = currentQuestion.value === id ? undefined : id;
+}
+</script>
+<template>
+  <div>
+    <div
+      class="text-center flex flex-col items-center justify-center h-[295px] gap-3 relative overflow-hidden"
+    >
+      <div class="hero hero-bg"></div>
+      <div
+        class="text-[56px] leading-[48px] text-skin-heading font-semibold space-x-2"
+      >
+        The power of
+        <ICPro class="h-[46px] inline" />
+      </div>
+      <div class="max-w-[290px] text-md leading-6">
+        Pick the plan that works best for your organization.
+      </div>
+      <div
+        class="flex border rounded-full p-1 items-center leading-[22px] bg-skin-bg"
+      >
+        <button
+          v-for="p in Object.keys(TURBO_PRICES) as SubscriptionLength[]"
+          :key="p"
+          :class="[
+            'rounded-full py-1 text-skin-link',
+            { 'bg-skin-active-bg': subscriptionLength === p }
+          ]"
+          @click="subscriptionLength = p"
+        >
+          <div v-if="p === 'yearly'" class="pl-3 pr-1 flex gap-1">
+            Annual
+            <div
+              class="bg-orange-300/20 border border-orange-300 rounded-full px-2 text-sm leading-[18px]"
+            >
+              -20%
```

Comment:
If regular price is $7200 (12 * $600) and on yearly we pay $6000 then it's $1200 off.
So it's $1200 off of $7200 regular price so -16.6% 😆

Reply:
Correct, we used to show discount % in the networks plan page: https://snapshot-git-network-fees-payment-page-snapshot.vercel.app/#/payment/network